src/test/features/response/routes/how-much-paid-claimant.ts

```diff
@@ -0,0 +1,153 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+import { checkAlreadySubmittedGuard } from './checks/already-submitted-check'
+
+import { Paths as ResponsePaths } from 'response/paths'
+import { ResponseType } from 'response/form/models/responseType'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+import { checkNotDefendantInCaseGuard } from './checks/not-defendant-in-case-check'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath = ResponsePaths.defendantHowMuchPaidClaimant.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
+
+const draftOverride = {
+  response: {
+    type: ResponseType.DEFENCE
+  }
+}
+
+describe('Defendant response: How much have you paid the claimant options', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'citizen')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+
+      context('when response not submitted', () => {
+        it('should return 500 and render error page when cannot retrieve claim', async () => {
+          claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.serverError.withText('Error'))
+        })
+
+        it('should redirect to response type page when response type is not full admission', async () => {
+          draftStoreServiceMock.resolveFind('response', { response: { type: ResponseType.PART_ADMISSION } })
```

Comment:
That (and name of the test case as well) is misleading, route does not check whether selected response is a part admission or full admission.

I guess you wanted to express that unless defence is selected there will be a redirect to response type page. If so please update test accordingly.

Please do the same for similar scenario handled by POST handler.